packages/seacas/libraries/exodus/src/ex_get_assemblies.c

```diff
@@ -17,6 +17,8 @@
  */
 int ex_get_assemblies(int exoid, ex_assembly *assembly)
 {
+  EX_FUNC_ENTER();
```

Comment:
These aren't allowed here since `ex_get_assembly` has a call to `EX_FUNC_ENTER()`.  This macro is used for the thread-safe build and locks a mutex, so if this call is here and also in `ex_get_assembly()`, it will deadlock.